### PR TITLE
script to check repo's name usage

### PR DESCRIPTION
**Pull request by S11001001**

### CODE REVIEW

File: scripts/check-repo-names.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euo pipefail
+
+SCRIPTNAME=${0##*/}
```
Comment (ray-roestenburg-da) [inline scripts/check-repo-names.sh:8]: maybe a comment that this extracts the filename from a full path :-)

**Review by ray-roestenburg-da — APPROVED** [review]
Thanks!

File: scripts/check-repo-names.sh (hunk 1 above)
Comment (ray-roestenburg-da) [reply scripts/check-repo-names.sh:8]: Wait, it is never used? probably remove this.

File: scripts/check-repo-names.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+rename_script="scripts/check-repo-names.sh"
```
Comment (ray-roestenburg-da) [inline scripts/check-repo-names.sh:10]: maybe you wanted to use `SCRIPTNAME` as an override for this?

File: scripts/check-repo-names.sh [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+function check_patterns() {
```
Comment (alexmatson-da) [inline scripts/check-repo-names.sh:12]: can we make this run against the splice code dump directly? This script can call `scripts/copy-to-splice.sh /tmp/splice-dump` or something and cd there

File: scripts/check-repo-names.sh (hunk 2 above)
Comment (stephencompall-DA) [reply scripts/check-repo-names.sh:8]: No, it was just leftover copypasta. Removed in 54d5cb818e2735bc18395a05bb33a0fa33585b2b

File: scripts/check-repo-names.sh (hunk 3 above)
Comment (stephencompall-DA) [reply scripts/check-repo-names.sh]: Sure, you can pass `-s path/to/sourcerepo` and it will do that as of 17f92a6f6f4f11d0da369ae8556ce81019a31e28.

**Pull request merged**